### DISCUSSION TITLE
The Number of Files shown in search result is wrong

### DISCUSSION THREAD

**Discussion by jukzi** [community/community#64172] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Example:
https://github.com/search?q=FileBufferModelManager+path%3A**%2FFileBufferModelManager.java&type=code&ref=advsearch

![image](https://github.com/community/community/assets/51790620/b980bc96-c22d-4158-9c75-4f4c3ffedc26)

Probably its some count distinct, while the search result actually shows all files.

**Comment by look** [comment, marked as answer]
Sorry for the confusion. The count is an estimate based on unique files. When there are not a lot of results, as is the case here, we expand duplicates which causes the discrepancy you're seeing.
